packages/core/content-releases/server/src/services/__tests__/scheduling.test.ts

```diff
@@ -0,0 +1,117 @@
+import { scheduleJob } from 'node-schedule';
+import createSchedulingService from '../scheduling';
+
+const baseStrapiMock = {
+  features: {
+    future: {
+      isEnabled: jest.fn().mockReturnValue(true),
+    },
+  },
+};
+
+jest.mock('node-schedule', () => ({
+  scheduleJob: jest.fn(),
+}));
+
+describe('Scheduling service', () => {
+  describe('set', () => {
+    it('should throw an error if the release does not exist', async () => {
+      const strapiMock = {
+        ...baseStrapiMock,
+        db: {
+          query: jest.fn(() => ({
+            findOne: jest.fn().mockReturnValue(null),
+          })),
+        },
+      };
+
+      // @ts-expect-error Ignore missing properties
+      const schedulingService = createSchedulingService({ strapi: strapiMock });
+      expect(() => schedulingService.set('1', new Date())).rejects.toThrow(
+        'No release found for id 1'
+      );
+    });
+
+    it('should cancel the previous job if it exists and create the new one', async () => {
```

Comment:
`it('should cancel the previous job if it exists and create the new one', async () =>`
to
`it('should cancel the previous job if exists and creates the new one', async () =>`